src/HtmlUtils.js

```diff
@@ -498,7 +501,17 @@ export function bodyToHtml(content, highlights, opts={}) {
     let emojiBody = false;
     if (!opts.disableBigEmoji && bodyHasEmoji) {
         EMOJI_REGEX.lastIndex = 0;
-        const contentBodyTrimmed = strippedBody !== undefined ? strippedBody.trim() : '';
+        let contentBodyTrimmed = strippedBody !== undefined ? strippedBody.trim() : '';
+
+        // Ignore spaces in body text. Emojis with spaces in between should
+        // still be counted as purely emoji messages.
+        contentBodyTrimmed = contentBodyTrimmed.replace(/\s/g, '');
```

Comment:
for consistency can haz this regex in a const created just once also plz

Reply:
okay i do

Reply:
And done :^)